helpers/seed.go

```diff
@@ -0,0 +1,114 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/gofrs/uuid"
+	models "github.com/layer5io/meshery/models"
+	"github.com/sirupsen/logrus"
```

Comment:
If we can, let's use meshkit logger here?
https://github.com/layer5io/meshkit/blob/master/logger/logger.go